README.md

```diff
@@ -91,12 +155,10 @@ end
 A composite cache is ideal for mitigating hot spot latency in frequently invoked areas of the codebase.
 
 ```ruby
-# method that's invoked frequently by multiple processes
+# method that's invoked frequently by multiple processes/machines
 def hotspot
-  # NOTE: expiration options are only applied to the outermost cache
-  #       inner caches use their globally configured expiration policy
-  Rails.composite_cache.fetch("example/slow/operation", expires_in: 12.hours) do
-    # a slow operation
+  Rails.composite_cache.fetch("example", expires_in: 12.hours) do
+    # computationally expensive operation with high latency...
```

Comment:
Very much agree with calling out high frequency access as an important consideration. 

I think that object size should definitely be explained but don't think it's a primary driver. For example, if you can only fit 10 objects in the local in-memory cache but those objects are accessed at a high frequency, you're still reaping some benefits, but it may be an indicator that you should reduce the size of cached objects or dedicate more memory. 

I mostly view factors like what, when, where to cache as requiring some common sense that everyone should be aware of when optimizing performance with a library like this.

Reply:
Yup, totally agree that's not primary. Re "reduce the size of cached objects" I think of that as picking more judiciously which objects use the special `CompositeCacheStore` vs. the general-purpose cache.